### DISCUSSION TITLE
Weird issue with copilot not working

### DISCUSSION THREAD

**Discussion by FC5570** [community/community#40370] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm using vscode. Copilot was working perfectly fine a few hours ago. It suddenly stopped giving suggestions. I clicked on the copilot icon in the status bar below, but nothing showed up, clicking on the icon just reloads all extensions.

When the copilot extension is enabled, JS/TS language features and other extensions fail to load and I get an error saying "extension host terminated unexpectedly". When I disable the extension, the issue goes away and everything loads as expected.

There are no logs in `Output -> Github Copilot` aswell. 

This is a recording of what happens:

https://user-images.githubusercontent.com/68158483/204576620-9ccd89c1-2902-4ae6-a53b-b7086c3986f1.mp4

I've cleared vscode cache, reinstalled the extension multiple times, and restart  vscode itself. I don't understand why this is happening.

**Comment by FollowTheProcess** [comment]
I'm getting this too

**Debug Info:**

```plaintext
Version: 1.74.0-insider
Commit: 8b9891739b703b50547f506a7e2bc9565e52beff
Date: 2022-11-29T07:10:26.814Z
Electron: 19.1.6
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin x64 22.1.0
Sandboxed: Yes
```

It offers to open developer tools after a while:

<img width="459" alt="image" src="https://user-images.githubusercontent.com/62767721/204631767-9c5b9032-c387-439e-8812-4d2c80502f70.png">

And when I do it looks like an internal V8 OOM error:

<img width="547" alt="image" src="https://user-images.githubusercontent.com/62767721/204631967-77332d54-26d5-4c8d-9ad1-86d40e0dfc5a.png">

I can confirm it's the copilot extension because I reloaded with all extensions disabled and re-enabled them one by one until I hit the error again, every other one was fine but re-enabling the copilot extension triggered the error.

I also installed a few other versions of the copilot extension going back to `1.57.7193` and it still happens so I'm not sure it's a recent change either.

Anyway hope this info helps get to a solution 👍🏻

**Comment by Fyko** [comment]
relavent https://github.com/community/community/discussions/40382

**Comment by iamcjwilliams** [comment]
@FC5570 @FollowTheProcess Our copilot team is investigating this issue

**Comment by FC5570** [comment]
The issue seems to have been fixed. Thanks.
